### PR TITLE
Forecast demo: new section

### PR DESCRIPTION
**Pull request by ryantibs**
I wrote a new section on how the forecast errors vary by the number of days ahead.

### CODE REVIEW

**Comment by ryantibs** [comment]
@capnrefsmmat Git complained to me when I pushed because the new RData file (forecast-demo/demo-extended.rda) is over 50 MB.  It's just barely over, it appears.  Do you think we should leave it in here or do you have an alternative suggestion?

**Comment by capnrefsmmat** [comment]
You could try using `save(..., compress = "xz")` to see if that makes the object smaller.

File: content/post/2020-09-21-forecast-demo.Rmd [hunk 1]
```diff
@@ -445,12 +445,84 @@ ggplot(res_dif2 %>%
...
+"Cases + Facebook" model have comparable median scaled error to 14-day-ahead 
+forecasts from the "Cases" model. 
+ 
 ## Wrap-Up
 
 We've seen that either of the Facebook or Google % CLI-in-community 
 signals can improve the accuracy of short-term forecasts 
 of county-level COVID-19 case incidence rates. 
-The statistical significance associated with these improvements 
+The significance associated with these improvements
```
Comment (capnrefsmmat) [inline content/post/2020-09-21-forecast-demo.Rmd]:
```suggestion
The significance of these improvements 
```

**Comment by ryantibs** [comment]
Thanks Alex, that worked (compression).

**Pull request merged**